### PR TITLE
Fabo/query keybase identities

### PR DESCRIPTION
**Pull request by faboweb**
Needs https://github.com/luniehq/lunie/pull/4415

Checks outdated profile pictures and adds them to the DB

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Why are there commits from #4235 here?

File: api/data/network-capabilities.js [hunk 1]
```diff
@@ -1,233 +0,0 @@
...
-    action_undelegate: capabilityEnum.ENABLED,
-    action_deposit: capabilityEnum.ENABLED,
-    action_vote: capabilityEnum.ENABLED,
-    action_proposal: capabilityEnum.ENABLED
-  }
-}
-
-module.exports = {
-  getNetworkCapabilities
-}
```
Comment (Bitcoinera) [inline api/data/network-capabilities.js:233]: :eyes:

File: api/changes/ana_control-network-enable-from-db [hunk 2]
```diff
@@ -0,0 +1 @@
+[Changed] [#4235](https://github.com/cosmos/lunie/pull/4235) Enabled field in network can be overwritten by its value on DB table "networks"  @Bitcoinera
```
Comment (Bitcoinera) [inline api/changes/ana_control-network-enable-from-db:1]: Good! Acknowledgement :wink:

**Comment by Bitcoinera** [comment]
So I guess this closes #4235

**Comment by Bitcoinera** [comment]
Testing this connection to DB gets lost, returning 502 and prompting this error:

```
newBlockHandler failed FetchError: invalid json response body at https://staging-db.lunie.io/v1/graphql reason: Unexpected token B in JSON at position 4
    at /home/ana/lunie/lunie/node_modules/node-fetch/lib/index.js:272:32
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async /home/ana/lunie/lunie/api/lib/database/helpers.js:5:16
    at async /home/ana/lunie/lunie/api/lib/database/helpers.js:119:15
    at async Object.getValidatorsInfo (/home/ana/lunie/lunie/api/lib/database/methods.js:60:10)
    at async BlockStore.addValidatorPictures (/home/ana/lunie/lunie/api/lib/stores/block-store.js:261:27)
    at async BlockStore.update (/home/ana/lunie/lunie/api/lib/stores/block-store.js:53:20)
    at async CosmosNodeSubscription.newBlockHandler (/home/ana/lunie/lunie/api/lib/block-listeners/cosmos-node-subscription.js:163:7) {
  type: 'invalid-json'
}
```

**Comment by faboweb** [comment]
This seems like a different error if you can't connect to the db at all.

**Comment by faboweb** [comment]
I merged the networks PR in here

File: scriptRunner/src/keybase.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            updateableValidators
+            .map(async ({ operator_address, profile_identifier, name }) => {
+                const query = `https://keybase.io/_/api/1.0/user/user_search.json?q=${profile_identifier}&num_wanted=1`
+                const result = await fetch(query).then(res => res.json())
+                if (result.list.length > 0) {
+                    return {
+                        name,
+                        operator_address,
+                        picture: result.list[0].keybase.picture_url
+                    }
```
Comment (Bitcoinera) [inline scriptRunner/src/keybase.js:32]: Is it not possible to just store the images in DB? Or are they so dynamic? Don't think so, right?

Comment (Bitcoinera) [reply scriptRunner/src/keybase.js:32]: ah ok, only update if we see they updated their profile. Nice, slick!

Comment (faboweb) [reply scriptRunner/src/keybase.js:32]: What is the problem you want to solve?

File: api/lib/apollo.js [hunk 4]
```diff
@@ -42,14 +42,17 @@ async function checkIsValidIdToken(idToken) {
...
+
   if (config.env !== 'test') {
-    startBlockTriggers()
+    startBlockTriggers(networks)
   }
 
   let options = {
     typeDefs,
-    resolvers,
+    resolvers: resolvers(networks),
```
Comment (Bitcoinera) [inline api/lib/apollo.js:55]: :tada:

**Comment by Bitcoinera** [comment]
:sunglasses: 

image

**Comment by Bitcoinera** [comment]
> This seems like a different error if you can't connect to the db at all.

Yes, strange. Then it just didn't happen again

**Pull request merged**